### PR TITLE
Dockerization

### PR DESCRIPTION
**Pull request by Moliholy**
This PR "dockerizes" the monitor, so that it adds the sufficient scripts and files to launch the docker container in "almost" a single instruction. The only requirement is to change settings.py.example accordingly if one wishes to use also the cvmfs_browser (disabled by default).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM centos:7
+MAINTAINER Jose Molina <jose.molina@cern.ch>
+
+RUN yum install epel-release -y
+RUN yum install httpd python-pip gcc git swig mod_wsgi openssl wget python-devel openssl-devel -y
+RUN pip install --upgrade pip
```
Comment (reneme) [inline Dockerfile]: Might be a superstition, but I read that it is a good idea to group multiple `RUN` statements into a single one. This produces less image layers and is supposed to gain some performance.

File: cvmfs_web/settings.py.example [hunk 2]
```diff
@@ -181,24 +182,26 @@ LOGGING = {
...
 # Number of tags that will be visualized per page
-# CLOUD_BROWSER_DEFAULT_LIST_LIMIT = 20
+CLOUD_BROWSER_DEFAULT_LIST_LIMIT = 20
 
 # change this accordingly and DO NOT FORGET TO CREATE THE DIRECTORY BEFORE.
 # Please also note this cache will contain the uncompressed files that are
 # downloaded upon user request and so far it won't be deleted (unless
 # manually, of course).
-# CLOUD_BROWSER_CVMFS_CACHE = os.path.join(BASE_DIR, 'cvmfs_cache')
+CLOUD_BROWSER_CVMFS_CACHE = '/var/www/html'
```
Comment (reneme) [inline cvmfs_web/settings.py.example]: Why in `/var/www`? Shouldn't the cache be somewhere outside of Apache's reach?

Comment (Moliholy) [inline cvmfs_web/settings.py.example]: One might have to serve these files when the user requests them, so they got to be in an accessible directory.
/var/www/html is accessible by default in apache in any configuration, and well known to change the proprietary to the "apache" user. If one wants to set another directory it would be necessary that that folder belongs to the user apache and granting permissions in the .conf file, making more difficult its access.

To sum up, because it's easy to set a well-known default value that works independently on any apache configuration

**Comment by Moliholy** [comment]
Changes:
- Cache moved from /var/www/html to /var/lib/cvmfs. Permissions granted to that directory in the Dockerfile
- Improved Dockerfile executing longer commands and thus decreasing the number of layers
- Change the URL system. When developing the Android application I realized it wasn't working for all the different cases, so now it directly maps FQRN --> URL so that there is no room for errors.

However, it's also noticeable that this URL mapping is only provisional. When the python utils have the proxy feature it should be immediately removed.
